pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1410,7 +1413,7 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       if (await this.isDistroRegistered({ distribution: DATA_INSTANCE_NAME })) {
         await this.execWSL('--unregister', DATA_INSTANCE_NAME);
       }
-      this.cfg = undefined;
+      this.cfg = defaultSettings;
```

Comment:
See line 137 above — this should be `undefined` here. (It should also be a separate commit.)